### DISCUSSION TITLE
I don't see stars and fork given on my repos anymore

### DISCUSSION THREAD

**Discussion by gracicot** [community/community#66165] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

With the old feed, I could see stars given to my repos, now we only stars given by people I follow to other repos. Was this intended to remove the information from the feed?

**Comment by slembcke** [comment]
Yeah... "Are people using my thing?", and "How are people using my thing?" is about the only bit of information I want to know from the feed. Yet there is no option to see if people star or fork things. -_- Instead it's a social media graph.

**Reply by gracicot** [reply]
Exactly. Sometime I see C# people starring my C++ repo, sometimes beginners, sometimes advanced people. Seeing who stars my repo help me building a knowledge of what's my user base.

**Reply by wangrongding** [reply]
# Github Old Feed  

💪 Using this script you can restore the old version of the feed.
https://github.com/wangrongding/github-old-feed  

<table>
  <tr><td width="50%" align="center"><b>Before (the shit💩 new feed)</b></td></tr>
  <tr>
     <td>
        <p align="center">
             <img width="800" align="center" src="https://assets.fedtop.com/picbed/1694066442147.jpg"/>
        </p>
     </td>
  </tr>
  <tr><td width="50%" align="center"><b>After use (the old one)</b></td></tr>
  <tr>
     <td>
        <p align="center">
             <img width="800" align="center" src="https://assets.fedtop.com/picbed/1694066437621.jpg"/>
        </p>
     </td>
  </tr>
</table>

**Comment by abb128** [comment]
It seems an unintentional glitch, given a filter for it exists.
![image](https://github.com/community/community/assets/65567823/b7476ce6-d31a-4abe-b3d9-3d837fb1b119)

**Reply by tidwall** [reply]
That **does not** show when people star your project. It only shows when people star other projects.

**Reply by slembcke** [reply]
Exactly... I'm not interested in another social media platform. I just want a dashboard that tells me how my repositories are being used. Honestly it would be great if it went even further in that direction and put information from the "insights" tab there since the stars are kinda arbitrary.

**Reply by RockChinQ** [reply]
I've checked it, but still cannot see my repo being starred😐

**Reply by gracicot** [reply]
This does not address the use case I'm talking about. This is not a solution. This important feature from the old feed is still not present when that checkbox is checked.

**Reply by dmnsgn** [reply]
And we need both, as separate options: seeing when our own repos are being starred and when people we follow star repos.

**Reply by AndyObtiva** [reply]
> That does not show when people star your project. It only shows when people star other projects.

That is exactly the problem! It means GitHub lied by telling us we get a consolidated feed as we lost information from the older feed. Bring back the "who starred your projects" feed as an extra option that is enabled by default for everyone to keep compatibility with the old feed. People can customize and remove if they don't need it. That pleases everybody 100%.

**Reply by nukeop** [reply]
Is there a single person on the planet that wants to see random projects getting stars, posting releases, or getting informed about issues being opened in random repos you've never seen?

**Comment by petarov** [comment]
Came here to open exactly an issue like this one. GH, please fix!

**Comment by 9001** [comment]
Looks like the old feed is still available for now: https://github.com/dashboard-feed

Not sure what I'll do when it goes away; the new feed is entirely useless to me.

**Reply by GiorgosXou** [reply]
Not for mobile displays sadly..

**Reply by GiorgosXou** [reply]
Also, The new layout looks so wrong on mobiles, the wrapping of  text in feed-posts looks awful

**Reply by ddalcino** [reply]
Thanks for the link! https://github.com/dashboard-feed is useful to me. The new default is of no use at all.

**Reply by hsfzxjy** [reply]
Thanks bro! The page makes me less anxious

**Reply by ris58h** [reply]
Thanks. It looks like a page for mobile devices though.

**Reply by mxmlnkn** [reply]
I also got pointed to https://github.com/Gerrit0/old-github-feed which works fine for me!

**Comment by shailshouryya** [comment]
Having a toggle to choose which feed (the previous, "simple" feed versus the new, algorithmic feed) would be great - perhaps making the https://github.com/dashboard-feed mentioned above more findable would be a decent starting point.

EDIT: made https://github.com/orgs/community/discussions/66226 to gather recent discussions about this new feed

**Comment by sandarutharuneth** [comment]
About say the same thing....
@github Please roll back the new changes to the feed

**Comment by ris58h** [comment]
+1

**Comment by wangrongding** [comment]
# Github Old Feed  

💪 Using this script you can restore the old version of the feed.
https://github.com/wangrongding/github-old-feed  

<table>
  <tr><td width="50%" align="center"><b>Before (the shit💩 new feed)</b></td></tr>
  <tr>
     <td>
        <p align="center">
             <img width="800" align="center" src="https://assets.fedtop.com/picbed/1694066442147.jpg"/>
        </p>
     </td>
  </tr>
  <tr><td width="50%" align="center"><b>After use (the old one)</b></td></tr>
  <tr>
     <td>
        <p align="center">
             <img width="800" align="center" src="https://assets.fedtop.com/picbed/1694066437621.jpg"/>
        </p>
     </td>
  </tr>
</table>

**Comment by palash25** [comment]
>  now we only stars given by people I follow to other repos

i don't even see that, all i see is people who followed me i don't see what other people are starring, i follow around 300 accounts and my feed used to be very active but now i see nothing.

**Reply by gracicot** [reply]
I see stars given by people I sponsor, but not people I follow.

**Comment by FreakyAli** [comment]
Can we please role this back? I don't care what other people are doing to be honest i just want to know the people that are using my repos...

**Comment by RainOrigami** [comment]
Ah yes, the classic Microsoft regression. Update a feature, make it less useful. Thanks!

**Reply by FreakyAli** [reply]
The one good thing about Github's dashboard is now gone, its soo bloody annoying

**Comment by denverharry** [comment]
you can see stars on my [Asheville concrete company](https://www.ashevilleconcretecontractors.com/).

**Comment by hewiefreeman** [comment]
This reminds me of LinkedIn killing itself by turning into a social media platform. Good to see another useful product holding a gun to it's own head.

**Reply by leeoniya** [reply]
don't worry, as has been joked elsewhere in these threads, LinkedIn integration is the next logical step of the roadmap.

**Comment by SaschaWillems** [comment]
Just want my voice to be heard, so I'm leaving a comment in here too. You took the only interesting thing from my feed/landing page, making that awkward landing page even less useful than it already has been.

Maybe you should start listening to your user base before going live with such changes.

Why not ask us (the users) what we want to see on our landing page instead of (seemingly) randomly changing things for the worse?

**Reply by Drarig29** [reply]
+1

**Reply by leeoniya** [reply]
> Why not ask us (the users) what we want to see on our landing page instead of (seemingly) randomly changing things for the worse?

because they demonstrably don't give a shit.

this is a carbon copy of the identical fiasco from 10 months ago, with all the same "feedback" provided to them, in great volume. with this latest, literal  ":middle_finger: fuck you, users :middle_finger:", it is clear that they have no interest in engaging in good faith, or engaging at all.

i've submitted detailed feedback/advice to them previously on maybe 5 separate occasions over the past 10 years and each was met with some cookie-cutter, "Thanks" reply, then radio silence, and nothing ever done about anything. listening to their users is not something they know how to do.

**Comment by beetaa** [comment]
a terrible update

**Comment by kriswiner** [comment]
Yup, not seeing any stars that were just there the other day....something is broken...

**Comment by orchetect** [comment]
I want to see how people are interacting with my repos. The old feed was great and showed exactly what I wanted to see. The new one is 100% useless.

**Comment by whitews** [comment]
Like many here, I found it useful to see how people were using my work via starring my repos. Showing activity from people I follow was also useful, as a colleague or friend that starred a repo might be relevant or interesting to me. Anything beyond that quickly becomes a distraction and a potential time-consuming rabbit hole...i.e. I come to GitHub to do work then get distracted by something completely unrelated to that. 

Version control is a tool to do a job. I don't need my woodworking tools to integrate with Instagram. I value knowing if people are using the products I build.

**Comment by kevinmoran** [comment]
The only thing I care about is people starring/forking/otherwise interacting with my repos, please fix

**Comment by AndyObtiva** [comment]
Same here. Only a week ago, I was seeing everyone who starred my repos (any people, not just ones I follow). Now, with the Stars filter only, all I see is this one star by someone I follow, who didn't even star one of my projects.

<img width="799" alt="Screenshot 2023-09-11 at 1 50 46 PM" src="https://github.com/community/community/assets/23052/73ec32a1-cf68-4e89-a818-ff3502e001a4">

My new feed doesn't show people who star my projects. I need that to know how my projects are trending. **It seems like the Stars filter only shows people I follow who star other projects, which is different**. Please bring this missing feature back, potentially with an option that could be selected in the filter if some people don't want to see which of their projects are starred by others.

I am not sure why GitHub removed features supported by the old feed instead of supporting all its features 100% via options while adding new features on top of that if needed. That's Software Product Design 101! GitHub did follow that principle when they added the For You Recommendations feed a while ago by keeping the old feed available. They should have simply done the same thing again when they released additional feed features. 

Make the exact same old feed available again through options.

**Comment by Jalle19** [comment]
My feed is now empty because I tried to filter out everything except who stars my projects. Came here after wondering if there's something wrong with the feed. An exceptionally stupid update...

**Comment by romis2012** [comment]
The same issue, stars and forks are not displayed on my feed

**Comment by lisenet** [comment]
Stars and forks are no longer displayed on the feed rendering it useless.

**Comment by nukeop** [comment]
Who asked for a feed of random events happening in random repositories you've never expressed any interest in? Why would I care that someone uploaded some binaries or opened an issue in a random repository? Was there a single person that requested this feature, or was this born in the mind of some MBA at Microsoft?

**Comment by FreakyAli** [comment, marked as answer]
With the below options selected things seem to go back to normal! 

Good to know things were rolled back!

![Screenshot_2023-09-14-13-48-03-25_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/community/community/assets/31090457/d549f1db-c39f-47a3-a659-e27f78c52761)

**Reply by Jalle19** [reply]
Indeed, stars show up in my feed now without me changing any settings!

**Reply by gracicot** [reply]
Yup. I see people starring my repos again. I'm not sure about forks though, but that fine.

**Reply by tidwall** [reply]
**Not back to normal for me.**

I just checked and people who star my projects are still not appearing on the new feed.

Here's what I'm seeing:

---

### New Feed

<img width="891" alt="image" src="https://github.com/community/community/assets/1156077/04398680-4943-4ccf-857c-601ebb55dbc2">

---

### Old feed

*using the old url https://github.com/dashboard-feed*

<img width="664" alt="image" src="https://github.com/community/community/assets/1156077/a20ebb92-216d-4de2-8ff9-87aec5c75743">

---

**Reply by FreakyAli** [reply]
@tidwall I think it has something todo with the options you have selected, only select what you see selected on my post earlier

**Reply by tidwall** [reply]
@FreakyAli 

Still nothing.

<img width="866" alt="image" src="https://github.com/community/community/assets/1156077/7545c17d-afa9-48d9-a49f-c11327152673">

**Reply by FreakyAli** [reply]
That's really weird it worked for alot of my friends and myself, you might wanna clear cache once. If not may be raise an issue

**Reply by gracicot** [reply]
This is what I see now. This is good enough I guess? As long as I see the new information with the new stuff I will consider it solved.

<img width="821" alt="Screenshot 2023-09-14 at 11 21 21 AM" src="https://github.com/community/community/assets/2906673/0988abec-68ba-4954-92a0-8eeaf15b4dfe">

**Reply by tidwall** [reply]
It's good that it's now working for some people. Unfortunately it's still not working for me. My old [dashboard feed](https://github.com/dashboard-feed) shows dozens of stars in the last few days, yet none of those show up on my new feed. Perhaps Github is rolling out the fix in stages, where some folks are starting to see sooner than others. But I give up trying at this point. I unchecked all the boxes in the new feed to quiet the noise and just use the https://github.com/dashboard-feed as my go to. At least until that disappears too.

Grin and bear it, as they say.

**Reply by cuducos** [reply]
I used to work here (like, this week), but It's broken now 💩 

Just as @tidwall said:
* yes, I have the _Starts_ option selected
* there are stars on the [old feed](https://github.com/dashboard-feed) (aka the users like better)
* there are no starts on the current _official_ feed

**Reply by woheller69** [reply]
It worked for 1 day a month ago.

**Comment by kriswiner** [comment]
Working for me now...

**Reply by woheller69** [reply]
I see some stars but not all of them. The last 2 days are missing and forks created aren't shown either

**Reply by woheller69** [reply]
Still does not work

**Comment by marakew** [comment]
all forks still missed in my feed

**Reply by FreakyAli** [reply]
I can see my forks interestingly

**Comment by nukeop** [comment]
I don't see my repos getting starred either anymore. It looks like the feed stopped updating a few weeks ago, entirely.

**Reply by cuducos** [reply]
[Same here](https://github.com/orgs/community/discussions/66165#discussioncomment-7283122)
